cmd/cluster-node-tuning-operator/main.go

```diff
@@ -258,6 +265,64 @@ func removePerformanceOLMOperator(cfg *rest.Config) error {
 	return nil
 }
 
+// During upgrade from 4.12 -> 4.13, this logic will update the authoritative flag in the new
+// Infrastructures.Status.CPUPartitioning to migrate Single Node clusters to the new method.
+// TODO: Revisit after 4.13 to remove logic when no longer needed.
+func migratePinnedSingleNodeInfraStatus(cfg *rest.Config, scheme *apiruntime.Scheme) error {
```

Comment:
should this PR be targeted at release-4.13 instead, as 4.13 and master have already branched? the master one could not have these pieces, and then the 4.13 one could, ensuring a correct transition.

Reply:
Oh boy, I'm not sure actually @jmencak do you have any thoughts on this? I was under the impression the branching was happening next week Friday?

Reply:
We already have release-4.13 and release-4.14 branches, **but**, they are still [frozen for merging](https://github.com/openshift/cluster-node-tuning-operator/issues/97) (i.e. read-only for us) and are [being fast-forwarded to from the master branch every hour](https://docs.ci.openshift.org/docs/architecture/branching/).  FWIU, what will happen on 3rd March unless the date is pushed out is that:

- we stop fast-forwarding master to release-4.13 branch ("feature-complete" == "feature-freeze" && "code-freeze" in 4.13, i.e. 0 days between ff & cf)
- release-4.14 branch opens for development (master will defacto become 4.14 -- it will continue being fast-forwarded to 4.14)

Reply:
That's great, we should be good then, resolving this.